BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/DiagnosticsModule/DiagnosticsEventListener.cs

```diff
@@ -81,5 +80,39 @@ protected override void OnEventSourceCreated(EventSource eventSource)
 
             base.OnEventSourceCreated(eventSource);
         }
+
+        /// <summary>
+        /// This method checks if the given EventSource Name matches known EventSources that we want to subscribe to.
+        /// </summary>
+        private static bool ShouldSubscribe(EventSource eventSource)
+        {
+            if (eventSource.Name.StartsWith("Microsoft-A", StringComparison.Ordinal))
+            {
+                switch (eventSource.Name)
+                {
+                    case "Microsoft-ApplicationInsights-Core": 
+                    case "Microsoft-ApplicationInsights-Data": 
```

Comment:
Microsoft-ApplicationInsights-Data -- what is this one?

Reply:
Here: https://github.com/microsoft/ApplicationInsights-dotnet/blob/ca03bc6a7c154b3006d55fc2f5c2f0ead5e1dc57/BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/RichPayloadEventSource.cs

Reply:
We shouldn't collect it then. Its not for sdk logging. Its for profiler consumption.